packages/openfermion/quri_parts/openfermion/transforms/__init__.py

```diff
@@ -293,5 +340,28 @@ def get_state_mapper(
 
         return super().get_state_mapper(n_spin_orbitals, n_fermions)
 
+    def get_inv_state_mapper(
+        self,
+        n_spin_orbitals: int,
+        n_fermions: Optional[int] = None,
+        n_up_spins: Optional[int] = None,
+    ) -> QubitFermionStateMapper:
+        if n_fermions is None:
+            raise ValueError("n_fermions is required.")
+        if n_up_spins is None:
+            raise ValueError("n_up_spins is required.")
+        if 2 * n_up_spins - n_fermions not in [0, 1]:
+            raise ValueError("Current implementation only supports sz = 0.0 or 0.5.")
+        return super().get_inv_state_mapper(n_spin_orbitals, n_fermions, n_up_spins)
+
+    def augment_dropped_bits(
+        self,
+        bit_array: list[int],
+        n_spin_orbitals: int,
+        n_fermions: Optional[int] = None,
+    ) -> list[int]:
+        # Add two qubits dropped by the fermion-to-qubit mapping.
+        return bit_array + [0, 0]
+
```

Comment:
I'm wondering if this method should be public?
My concern is that for SCBK inverse mapping, we do not rearrange the dropped qubit back to the M/2-th and M-th qubit with this method (M is the number of spin orbitals.) Would this cause user's confusions if they decide to use it as a public method?

Reply:
Yeah, maybe we can make it private (or you can call it "protected", a term used in other programming languages like C++ or Java).

Reply:
I will make it protected in this PR